fee_calculator/apps/api/tests/test_advocate_type_api.py

```diff
@@ -6,7 +6,7 @@
 
 from calculator.constants import SCHEME_TYPE
 from calculator.models import Scheme
-
+from calculator.tests.lib.utils import prevent_request_warnings
 
```

Comment:
Mostly it's just that there should two blank lines between the final import statement and the first class or function definition. Like I say, super trivial.

Reply:
👍 thanks for this